api/_mixins/LinkStyle__HTMLLinkElement.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "HTMLLinkElement": {
+      "sheet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLLinkElement/sheet",
```

Comment:
This doesn't exist, at least not yet. Will you also prepare a PR for https://github.com/mdn/content that removes all references to `LinkStyle`, similar to https://github.com/mdn/content/pull/5579? https://github.com/mdn/content#moving-one-or-more-documents might come in handy if so.

Reply:
I haven't contributed to the content repository yet, but I'd be happy to write a PR to go along with this one!

Reply:
Apparently there's no page for `LinkStyle.sheet`, so I can't move/copy it.  Additionally, there aren't any pages for the `sheet` property on any of the inheriting APIs.  Not really sure what to do on the content repo, would love some assistance!

Reply:
I think the main task here is to remove mentions of `LinkStyle.sheet`. There's no page, but there are references to it. Searching that string should mostly turn up variations on `{{domxref("LinkStyle.sheet")}}`—there's just a few of them, but you'll need to read it in context to figure out what exactly to cut out (e.g., a `<dt>`-`<dd>` pair).

Reply:
I've sent https://github.com/mdn/content/pull/5669 to accompany this PR!